code/style.css

```diff
@@ -1,7 +1,339 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  margin: 0;
+  background: #E1D2CB;
+  font-family: 'Montserrat', sans-serif;
+}
+
+
+/* -------------------NAVIGATIONBAR--------------------------- */
+
+.navbar {
+  width: 100%;
+  font-size: 15px;
+  font-weight: bold;
+  background-color: whitesmoke;
+  opacity: 0.5;
+  position: relative; 
+  z-index: 15;
+  display: flex; 
+  object-fit: cover;
+  overflow: hidden;
+  justify-content: space-evenly;
+  align-items: center;
```

Comment:
I removed align items, float:right, object fit:cover with the inspector and couldnt see any difference on your navbar in mobile view. What is the code doing in this particular case? Might be my inspector tool not showing the right things.

Reply:
Same with overflow hidden